test/k8sT/Services.go

```diff
@@ -315,6 +315,49 @@ var _ = Describe("K8sServicesTest", func() {
 			testCurlFromPods("id=app2", tftpSVCURL, 10, 0)
 		})
 
+		SkipItIf(helpers.RunsWithKubeProxy, "Checks service on same node", func() {
+			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+				"skipSocketLB": "true",
+			})
+			defer DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+				"skipSocketLB": "false",
+			})
```

Comment:
My reaction when I see this makes me think that this whole `It()` belongs in its own `Context`. The rationale is that if we have to set up Cilium is a specific way, that's a sign that it should be in its own `Context`. Is it possible to consolidate this test with another `Context` and pass the skip flag to that setup function as well or do you think it deserves its own separation?

Reply:
I think consolidating them into a seperate `Context` makes a lot of sense! Thanks and done :)